app/views/govuk_publishing_components/components/_feedback.html.erb

```diff
@@ -0,0 +1,112 @@
+      <h3 class="pub-c-feedback__prompt-question">Is this page useful?</h3>
+
+      <%= link_to contact_govuk_path, {
+        class: 'pub-c-feedback__prompt-link pub-c-feedback__prompt-link--useful js-page-is-useful',
+        data: {
+          'track-category' => 'manualFeedbackForm',
+          'track-action' => 'ffYesClick'
+          },
+        } do %>
+          Yes <span class="visually-hidden">this page is useful</span>
+      <% end %>
+
+      <%= link_to contact_govuk_path, {
+        class: 'pub-c-feedback__prompt-link js-toggle-form js-page-is-not-useful',
+        data: {
+          'track-category' => 'manualFeedbackForm',
+          'track-action' => 'ffNoClick'
+          },
+          'aria-controls': 'page-is-not-useful',
+          'aria-expanded': false
+        } do %>
+          No <span class="visually-hidden">this page is not useful</span>
+      <% end %>
+
+      <%= link_to contact_govuk_path, {
+        class: 'pub-c-feedback__prompt-link pub-c-feedback__prompt-link--wrong js-toggle-form js-something-is-wrong',
+        data: {
+          'track-category' => 'manualFeedbackForm',
+          'track-action' => 'ffWrongClick'
+          },
+          'aria-controls': 'something-is-wrong',
+          'aria-expanded': false
+        } do %>
+          Is there anything wrong with this page?
+      <% end %>
+    </div>
+
+    <div class="pub-c-feedback__prompt-success js-prompt-success js-hidden" tabindex="-1">
+      Thank you for your feedback
+    </div>
+  </div>
+
+  <form action="/contact/govuk/problem_reports" id="something-is-wrong" class="pub-c-feedback__form js-feedback-form js-hidden" data-track-category="manualFeedbackForm" data-track-action="ffFormSubmit">
+    <a href="#" class="pub-c-feedback__close pub-c-feedback__js-show js-close-form" aria-controls="something-is-wrong" aria-expanded="true" role="button">Close</a>
+
+    <div class="pub-c-feedback__grid-row">
+      <div class="pub-c-feedback__column-two-thirds">
+        <div class="pub-c-feedback__error-summary pub-c-feedback__js-show js-hidden js-errors" tabindex="-1"></div>
+
+        <input type="hidden" name="url" value="<%= request.original_url -%>">
+        <input type="hidden" name="user_agent" value="<%= request.user_agent -%>">
+
+        <h2 class="pub-c-feedback__form-heading">Help us improve GOV.UK</h2>
+        <p id="feedback_explanation" class="pub-c-feedback__form-paragraph">Don’t include personal or financial information like your National Insurance number or credit card details.</p>
+
+        <%= render "govuk_publishing_components/components/input", {
+          label: {
+            text: "What were you doing?"
+          },
+          name: "what_doing"
+        } %>
+
+        <%= render "govuk_publishing_components/components/input", {
+          label: {
+            text: "What went wrong?"
+          },
+          name: "what_wrong"
+        } %>
+
+        <input class="pub-c-feedback__submit" type="submit" value="Send">
```

Comment:
Maybe this needs a `TODO` comment saying that we'll use the button component once it's moved to the gem.

Reply:
There's one in the description in the YML file, but I'll update it to be more explicit.